en/Getting started/Download and install Obsidian.md

```diff
@@ -27,7 +27,7 @@ If you use Linux, you can install Obsidian in several ways. Follow the instructi
 4. In the terminal, run the following command to install the Snap package:
```

Comment:
```suggestion
4. In the terminal, run the following command to install the Snap package: (the `--dangerous` flag is required because Canonical, the company who created Snap, did not review our package, the `--classic` flag allows Obsidian to access outside of the sandbox, where your notes are stored)
```